Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG MODE=normal
-ARG BASE_IMAGE=newrelic/infrastructure-bundle:2.8.28
+ARG BASE_IMAGE=newrelic/infrastructure-bundle:2.8.32
```

Comment:
This adds to this integration the issue of `debug` being moved to `trace`.

Nothing that we can fix but a thing to have in mind with other open issues about the `v2` version...